src/CMakeLists.txt

```diff
@@ -15,14 +16,19 @@ source_group(TREE ${CMAKE_SOURCE_DIR} FILES ${NOVELRT_LIB_SOURCES})
 set(NOVELRT_LIB_LINK_LIBRARIES
   Freetype::Freetype
   glad::glad
+  glfw
   glm
-  SDL2::SDL2-static
-  SDL2::SDL2_image
-  SDL2::SDL2_mixer
-  SDL2::SDL2main
+  OpenAL::OpenAL
+  png
   spdlog::spdlog
 )
 
+if (WIN32)
+  set(NOVELRT_LIB_LINK_LIBRARIES ${NOVELRT_LIB_LINK_LIBRARIES} sndfile-shared)
+else()
+  set(NOVELRT_LIB_LINK_LIBRARIES ${NOVELRT_LIB_LINK_LIBRARIES} sndfile-static)
```

Comment:
Why statically link on Unix?

Reply:
This is actually a requirement of libsndfile (used for reading the different file formats). I'd need to look into it more, but vcpkg specifies to use static linking on Unix - dynamic linking fails (it caused a CI failure for the Ubuntu checks prior).